### PR TITLE
Documentation for onBarcodeFrameAvailable callback and shouldPassBarc…

### PR DESCRIPTION
**Pull request by TomekJurkowski**
…odeFrame flag added.

### CODE REVIEW

File: docs/Scandit.java [hunk 1]
```diff
@@ -345,6 +345,12 @@
     */
     public ScanSettings scanSettings;
 
+    /**
+     *  @brief Prop used to inform the picker whether it should make the frame data available
+     *  via {@link BarcodePicker.onBarcodeFrameAvailable} callback.
```
Comment (petradonka) [inline docs/Scandit.java]:
```suggestion
     *  via the {@link BarcodePicker.onBarcodeFrameAvailable} callback.
```

File: docs/Scandit.java [hunk 2]
```diff
@@ -365,6 +371,16 @@
     */
     public function onSettingsApplied;
 
+    /**
+     *  @brief Prop used to set a callback that provides frame data in base 64 string format.
```
Comment (petradonka) [inline docs/Scandit.java]:
```suggestion
     *  @brief Prop used to set a callback that provides frame data in base64 string format.
```

File: docs/Scandit.java [hunk 3]
```diff
@@ -365,6 +371,16 @@
...
+     *
+     *  The onBarcodeFrameAvailable method is executed after each frame processed by the picker,
```
Comment (petradonka) [inline docs/Scandit.java]:
```suggestion
     *  The onBarcodeFrameAvailable method is executed after each frame is processed by the picker,
```

File: docs/Scandit.java [hunk 4]
```diff
@@ -365,6 +371,16 @@
...
+     *  but only if new codes have been recognized in that particular frame.
+     *  Note that the callback will only be invoked,
```
Comment (petradonka) [inline docs/Scandit.java]:
```suggestion
     *  Note that the callback will only be invoked
```

Comment (TomekJurkowski) [reply docs/Scandit.java]: done

File: docs/Scandit.java (hunk 3 above)
Comment (TomekJurkowski) [reply docs/Scandit.java]: done

File: docs/Scandit.java (hunk 2 above)
Comment (TomekJurkowski) [reply docs/Scandit.java]: done

**Pull request merged**